src/main/java/com/airlines/model/Crew.java

```diff
@@ -0,0 +1,44 @@
+package com.airlines.model;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+public class Crew {
+    private int id;
+    private String name;
+    private List<CrewMember> crewMembers;
+
+    private Crew() {
```

Comment:
What point of exising this constructor?

Reply:
remove in https://github.com/pielena/airlines-jdbc-app/commit/bc4e257c81648513d3a0f4bc5de1c6e21696c76a